app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileEditActivityTest.kt

```diff
@@ -387,6 +387,20 @@ class ProfileEditActivityTest {
     }
   }
 
+  @Test
+  fun testProfileEditActivity_configurationChange_startActivityWithUserProfile_clickProfileDeletionButton() { // ktlint-disable max-line-length
+    ActivityScenario.launch<ProfileEditActivity>(
+      ProfileEditActivity.createProfileEditActivity(
+        context,
+        1
+      )
+    ).use {
+      onView(withId(R.id.profile_delete_button)).perform(click())
+      onView(isRoot()).perform(orientationLandscape())
+      onView(withText(R.string.profile_edit_delete_dialog_positive)).check(matches(isDisplayed()))
```

Comment:
here, `profile_edit_delete_dialog_positive` is inside the dialog, I guess so night need to use inRoot(isDialog()).

Reply:
thanks. updated